server/src/main/java/org/apache/iotdb/db/qp/constant/SQLConstant.java

```diff
@@ -170,7 +170,13 @@ private SQLConstant() {
   public static final int TOK_QUERY_PROCESSLIST = 97;
   public static final int TOK_KILL_QUERY = 98;
 
-  public static final int TOK_CHILD_NODES = 99;
```

Comment:
You change it from 99 to 104, may this cause wal inconsistent problem? More specifically if token 99 has written into wal and I update the IoTDB, may this cause a problem?

Reply:
TOK_CHILD_NODES was added into master two days ago. So I don't think it's a big problem.
But it's still better to keep TOK_CHILD_NODES = 99.

Reply:
Thank you all. I'll keep TOK_CHILD_NODES = 99.